src/components/SearchResults.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Masonry, { ResponsiveMasonry } from "react-responsive-masonry";
+import { v4 as uuidv4 } from "uuid";
+import PropTypes from "prop-types";
+import "../styles/search-results.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p className="no-results">No results</p>;
+  }
+
+  return (
+    <div className="search-results" data-testid="search-results">
+      <ResponsiveMasonry columnsCountBreakPoints={{ 600: 1, 900: 2, 1200: 3 }}>
```

Comment:
I see what you went for with the use of the external library for styling, however in a tech test its generally better to avoid using styling libraries like this as the point is to display your understanding of the fundamentals and how you could create this same experience yourself